freqtrade/data/dataprovider.py

```diff
@@ -442,7 +444,24 @@ def refresh(self,
         if self._exchange is None:
             raise OperationalException(NO_EXCHANGE_EXCEPTION)
         final_pairs = (pairlist + helping_pairs) if helping_pairs else pairlist
+        # refresh latest ohlcv data
         self._exchange.refresh_latest_ohlcv(final_pairs)
+        # refresh latest trades data
+        self.refresh_latest_trades(final_pairs)
```

Comment:
so your suggestion is changing it to ` self.refresh_latest_trades(pairlist)`. or would you want it to configurable by itself by creating it's own pairlist in configuration?

Reply:
well my suspicion is that fetching it for both pairlist and informative_pairs will be wrong (not least because pairs might not be unique, only the combination of pair/timeframe/candletype is).


the actual timeframe shouldn't matter for a `refresh_trades()` call - so getting `("BTC/USDT", "5m")` and `("BTC/USDT", "15m")` would lead to duplicate work (it'll be the same trades you aim to fetch, the timeframe is really irrelevant here - at least after the initial call).

This part COULD be handled within the refresh call (but isn't IIRC).

---

Now if i need the informative BTC/USDT trades data if that pair is not in the pairlist is a different topic - one i don't have a very strong opinion yet - but i think users should not be forced to that, as it can have quite severe performance implications.

i might want ETH/BUSD trades data - but not for all other informative pairs, so i think this should be split from regular informative pair handling.